index.js

```diff
@@ -10,7 +10,7 @@ const { resolve, dirname, join, relative } = require('path')
 const rpj = path => readFile(path, 'utf8')
   .then(data => readBinDir(path, normalize(stripUnderscores(parse(data)))))
   .catch(er => {
-    er.path = path
+    er.file = path
```

Comment:
```suggestion
    er.path = path
    er.file = path
```

Reply:
The cost of having it in both spots is pretty minimal, and the pattern throughout most of our stack is that `er.path` is The Thing that had the problem.  This also makes it semver minor rather than semver major.